lib/srv/db/mysql/autousers.go

```diff
@@ -527,7 +523,7 @@ func getCreateProcedureCommand(conn *clientConn, procedureName string) (string,
 const (
 	// procedureVersion is a hard-coded string that is set as procedure
 	// comments to indicate the procedure version.
-	procedureVersion = "teleport-auto-user-v1"
+	procedureVersion = "teleport-auto-user-v2"
```

Comment:
It is not required because we check if all the procedures are in place. Otherwise, the update would be necessary. I just kept the change as a good practice for updating the procedures. I don't have a strong opinion.

Reply:
sounds good